spec/statement.dd

```diff
@@ -1103,9 +1103,17 @@ $(GNAME StatementNoCaseNoDefault):
     $(GLINK ScopeBlockStatement)
 )
 
-        $(P $(EXPRESSION) is evaluated. The result type T must be
-        of integral type or $(CODE char[]), $(CODE wchar[]) or $(CODE dchar[]).
-        The result is
+        $(P $(I Expression) is evaluated.
+        If the type of the $(I Expression) is an enum,
```

Comment:
The rule to follow here is, anything that is not an English word must have some annotation other than plain text. The word "enum" is not in the dictionary, so it needs to be in code font.